lib/15_in_words.rb

```diff
@@ -0,0 +1,75 @@
+class Fixnum
+  def in_words
+    hash = {
+      0 => nil,
+      1 => "one",
```

Comment:
http://stackoverflow.com/questions/16621073/when-to-use-symbols-instead-of-strings-in-ruby